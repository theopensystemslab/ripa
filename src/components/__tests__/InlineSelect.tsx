import { cleanup, render } from "@testing-library/react";
import * as React from "react";

import SelectQuestion from "../InlineSelect";

afterEach(cleanup);

describe("Select Question Component", () => {
  it("should render snapshot", () => {
    const { asFragment } = render(<SelectQuestion />);
    expect(asFragment()).toMatchSnapshot();
  });
});
