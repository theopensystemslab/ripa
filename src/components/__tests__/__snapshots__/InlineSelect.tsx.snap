// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select Question Component should render snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiFormControl-root makeStyles-container-1"
  >
    <div
      class="MuiInputBase-root MuiInput-root makeStyles-formControl-7 MuiInputBase-formControl MuiInput-formControl"
    >
      <div
        aria-haspopup="listbox"
        aria-labelledby="mui-component-select-select"
        class="MuiSelect-root MuiSelect-select makeStyles-selectInput-3 MuiSelect-selectMenu makeStyles-root-2 MuiInputBase-input MuiInput-input"
        id="mui-component-select-select"
        role="button"
        tabindex="0"
      >
        <span>
          â€‹
        </span>
      </div>
      <input
        name="select"
        type="hidden"
        value=""
      />
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSelect-icon makeStyles-icon-6"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7.41 8.59L12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
